client/me/purchases/purchases-list-in-dataviews/purchases-data-field.tsx

```diff
@@ -57,22 +59,26 @@ function PurchaseItemRowStatus( props: {
 function PurchaseItemRowPaymentMethod( props: {
 	purchase: Purchases.Purchase;
 	translate: LocalizeProps[ 'translate' ];
+	isBackupMethodAvailable: boolean;
 } ) {
-	const { purchase, translate } = props;
+	const { purchase, translate, isBackupMethodAvailable } = props;
 
 	return (
```

Comment:
I am beginning to wonder if I really need all of these wrapped functions. I could just return this same statement in the render function for the `payment-method` field on line 175.

I started with this pattern, since it made sense when we were needing to get the site state. Now, the only column that needs this is the `product` column. The pattern is nice and predictable, but I am wondering if it is adding bloat now.

Reply:
I removed the wrapper function for now. If it is better to keep the datafields render function clean, I can re-add the wrapper function and move all logic to it.